admin/sql/create_models.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+INSERT INTO model (model, model_version, build_sha1, date, include) VALUES ('danceability', 'v2.1_beta1', 'v2.1_beta1-228-g260734a', now(), 't');
```

Comment:
By adding this before the migration we don't have to do a lookup for the id each time we do a insert into `highlevel_model`, we just have a lookup table in python. much faster